cdap-docs/developers-manual/source/building-blocks/datasets/fileset.rst

```diff
@@ -49,8 +49,8 @@ To create and use a FileSet in an application, you create it as part of the appl
       createDataset("lines", FileSet.class, FileSetProperties.builder()
         .setBasePath("example/data/lines")
         .setInputFormat(TextInputFormat.class)
-        .setOutputFormat(TextOutputFormat.class).build());
         .setOutputProperty(TextOutputFormat.SEPERATOR, ":")
+        .setOutputFormat(TextOutputFormat.class).build());
```

Comment:
What about instead fixing this with:
```
.setInputFormat(TextInputFormat.class)
.setOutputFormat(TextOutputFormat.class)
.setOutputProperty(TextOutputFormat.SEPERATOR, ":")
.build());
```
to maintain the pattern, and make future changes easier?